hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperBasedRepository.java

```diff
@@ -34,6 +36,17 @@ protected void ensureConnected() {
         }
     }
 
+    protected void ensurePathExists(String path) {
+        ensureConnected();
+        if (!pathExists(path)) {
+            try {
+                zookeeper.create().creatingParentsIfNeeded().forPath(path);
+            } catch (Exception e) {
```

Comment:
IOException is imported but generic Exception is caught here, which one do we want?